source/Core/Src/settingsGUI.cpp

```diff
@@ -531,19 +531,20 @@ static void displayPDNegTimeout(void) {
 
 static void displayUSBPDMode(void) {
   /*
-   * PD Mode
-   * 0 = Safe mode, no PPS, no EPR
-   * 1 = Default mode, tolerant + PPS + EPR
-   * 2 = Strict mode + PPS + EPR
+   * Supported PD modes:
+   *  DEFAULT,    1 = PPS + EPR + more power request through increasing resistance by 0.5 Ohm
```

Comment:
```suggestion
   *  DEFAULT,    1 = PPS + EPR + more power request through increasing resistance by 0.5 Ohm to offset losses in cabling + PCB etc
```

Reply:
Nice suggestion! I agree that the clear - the better, therefore I took some liberty to clarify it even a little bit more, let me show you...